client/lib/utils/coerce-values.js

```diff
@@ -1,3 +1,5 @@
+let coerceFormValues;
```

Comment:
Well it's also because of the refactor in this PR - why does it make more sense to have `coerceValue` call `coerceFormValues`?